install.sh

```diff
@@ -0,0 +1,635 @@
+#!/bin/bash
+export LANG=en_US.UTF-8
+set -e
+
+LOGO=$(cat <<'EOF'
+    ___       ___       ___       ___       ___       ___       ___       ___       ___       ___   
+   /\  \     /\  \     /\  \     /\__\     /\  \     /\  \     /\  \     /\  \     /\  \     /\__\  
+  /::\  \   /::\  \   /::\  \   /:| _|_    \:\  \   /::\  \    \:\  \   /::\  \   /::\  \   /:/ _/_ 
+ /::\:\__\ /:/\:\__\ /::\:\__\ /::|/\__\   /::\__\ /\:\:\__\   /::\__\ /::\:\__\ /:/\:\__\ /::- \__\\
+ \/\::/  / \:\:\/__/ \:\:\/  / \/|::/  /  /:/\/__/ \:\:\/__/  /:/\/__/ \/\::/  / \:\ \/__/ \;:;- ,- 
+   /:/  /   \::/  /   \:\/  /    |:/  /   \/__/     \::/  /   \/__/      /:/  /   \:\__\    |:|  |  
+   \/__/     \/__/     \/__/     \/__/               \/__/               \/__/     \/__/     \|__|  
+EOF
+)
+
+APP_NAME="agentstack"
+VERSION="0.3.5"
```

Comment:
we could potentially create a GH workflow to dynamically update this whenever a change is made to the version number in pyproject.toml